tests/org.codechecker.eclipse.rcp.unit.tests/resources/CodeChecker/bin/CodeChecker

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+import sys
+import argparse
+
+def checkers(args):
+    print(' alpha.clone.CloneChecker\n'+
+            ' unix.API')
```

Comment:
Please add a checker name from clang-tidy too, it has different naming convention.